src/core/trulens/core/app/base.py

```diff
@@ -1475,7 +1475,7 @@ def dummy_record(
                 calls.append(sample_call)
 
         return mod_record_schema.Record(
-            app_id=self.app_id,
+            app_version=self.app_version,
```

Comment:
How do we support the following in OSS?
App 1 -> v0, v1, v2, v3
App 2 -> v0, v1
App 3 -> v1, v2.

From snowflake we support this by saying, create a different schema for the app, and every version is in tables. But I don't see how the same will work with open source. Should we introduce app-id and app-version as two different concepts instead of this?

Reply:
`Tru` now supports an `app_name` argument, which in OSS, would define the database filename (`default.sqlite` > 'app_1.sqlite'). 

The thought is that in OSS, different apps are stored in different DBs. Only in Snowflake would that distinction be made at the schema-level however.